packages/react/.storybook/manager-head.html

```diff
@@ -47,6 +47,12 @@
   src="https://1.www.s81c.com/common/carbon-for-ibm-dotcom/tag/v1/latest/footer.min.js"
 />
 
+<script
```

Comment:
```suggestion
<!-- Tealium/GA Set up -->
<script type="text/javascript">
  window._ibmAnalytics = {
    settings: {
      name: 'CarbonReactStorybookv10',
      isSpa: true,
      tealiumProfileName: 'ibm-web-app',
    },
    onLoad: [['ibmStats.pageview', []]],
  };
  digitalData = {
    page: {
      pageInfo: {
        ibm: {
          siteId: 'IBM_' + _ibmAnalytics.settings.name,
        },
      },
      category: {
        primaryCategory: 'PC100',
      },
    },
  };
</script>
<script
```

I think this needs to also get added. I copied from the v11 storybook and just changed the name to be unique.

Reply:
Thank you!